control/ipv4.yaml

```diff
@@ -0,0 +1,114 @@
+openapi: 3.0.3
+info:
+  title: Control ipv4 models
+  description: >-
+    All control ipv4 schemas
+  version: ^0.0.0
+components:
+  schemas:
+    Action.Protocol.Ipv4:
+      description: >-
+        Actions associated with IPv4 on configured resources.
+      type: object
+      required:
+        - choice
+      properties:
+        choice:
+          type: string
+          x-enum:
+            ping:
+              x-field-uid: 1
+          x-field-uid: 1
+        ping:
+          $ref: "#/components/schemas/Action.Protocol.Ipv4.Ping"
+          x-field-uid: 2
+    Action.Response.Protocol.Ipv4:
+      description: >-
+        Response for actions associated with IPv4 on configured resources.
+      type: object
+      required:
+        - choice
+      properties:
+        choice:
+          type: string
+          x-enum:
+            ping:
+              x-field-uid: 1
+          x-field-uid: 1
+        ping:
+          $ref: "#/components/schemas/Action.Response.Protocol.Ipv4.Ping"
+          x-field-uid: 2
+    Action.Protocol.Ipv4.Ping:
+      description: >-
+        Request for initiating ping between multiple source and destination pairs.
+      type: object
+      properties:
+        requests:
+          description: >-
+            List of IPv4 ping requests.
+          type: array
+          items:
+            $ref: "#/components/schemas/Action.Protocol.Ipv4.PingRequest"
+          x-field-uid: 1
+    Action.Protocol.Ipv4.PingRequest:
+      description: >-
+        Request for initiating ping between a single source and destination pair.
+
+        For ping request, 1 IPv4 ICMP Echo Request shall be sent and wait for ping response to
+        either succeed or time out. The API wait timeout for each request shall be 300ms.
```

Comment:
The value 300ms was arbitrarily picked as currently we're not allowing users to pick any other ping parameters. There's only one ping packet sent as per the description (instead of multiple packages, which would entail description about whether it's serial or concurrent).

I have also marked this schema with `x-status: under-review` as the parameters to be provided for ping request is still TBD. Ideally the parameters needn't be mentioned in schema description.